src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/fiks/SoknadService.kt

```diff
@@ -227,12 +233,18 @@ class SoknadService {
         val vedleggsId = UUID.randomUUID().toString()
         var vedleggsInfo: JsonVedlegg? = null
         var sha512 = "dummySha512"
-        if (vedleggsJson != null && !vedleggMetadata.filnavn!!.contentEquals(ettersendelseFilnavn)) {
-            vedleggsInfo = vedleggsJson.vedlegg.firstOrNull { jsonVedlegg ->
-                jsonVedlegg.filer.any { it.filnavn!!.contentEquals(vedleggMetadata.filnavn) }
-            }
-            if (vedleggsInfo != null) {
-                sha512 = vedleggsInfo.filer.first { it.filnavn!!.contentEquals(vedleggMetadata.filnavn) }.sha512
+        if (vedleggsJson != null) {
+            if (vedleggMetadata.filnavn!!.contentEquals(ettersendelseFilnavn)) {
+                if(file != null) {
+                    ettersendelsePdfLager.add(fiksDigisosId, ettersendelseFilnavn, file.bytes)
+                }
+            } else {
```

Comment:
Er det noe rart med innrykk her? 
(vanskelig å se hvilken if elsen hører til. Mulig det bare er github)

Reply:
Jeg tar det tilbake. er bare jeg som ble forvirra :D

Reply:
Det er lov å være forvirra. (Og det ser riktig ut for meg.) Kunne sikkert vært skrevet bedre uansett.

Reply:
Ja, forsto det til slutt. Men litt mye nøsta if-er her, ja 😄 
Tenker litt på at det er endel mulige feilsituasjoner her som vi verken kaster exception på eller logger. (så vi vet ikke om de inntreffer)
Greit nok for mock, vi kan heller legge på litt mer logging og vi føler systemet oppfører seg rart eller får tilbakemelding på at noe mangler 👍🏻